### PR TITLE
Add sample cli for sheets api

### PR DESCRIPTION
**Pull request by tknzk**
sample for sheets api

### CODE REVIEW

File: samples/cli/lib/samples/sheets.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      puts result.inspect
+      data = []
+      rows = result.values.length
+      rows.times.each do |row|
+        data.push(result.values[row])
+      end
+      print_table(data)
+    end
+
+    default_task :get_vlaues
```
Comment (codealchemy) [inline samples/cli/lib/samples/sheets.rb]: Typo here

Comment (tknzk) [inline samples/cli/lib/samples/sheets.rb]: fix typo. https://github.com/google/google-api-ruby-client/pull/417/commits/3a0d4b95303a568f03c40effe87acb4f6cb795bb

**Comment by sqrrrl** [comment]
Thanks :)

File: samples/cli/lib/samples/sheets.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    Sheets = Google::Apis::SheetsV4
+
+    desc 'get_values', 'Get values from Google Spreadseets '
+    method_option :spreadsheet_key, type: :string, required: true
+    method_option :range, type: :string, required: true
+    def get_values
+      sheets = Sheets::SheetsService.new
+      sheets.authorization = user_credentials_for(Sheets::AUTH_SPREADSHEETS_READONLY)
+      result = sheets.get_spreadsheet_values(options[:spreadsheet_key], options[:range])
+      puts result.inspect
```
Comment (sqrrrl) [inline samples/cli/lib/samples/sheets.rb]: needed?

Comment (tknzk) [inline samples/cli/lib/samples/sheets.rb]: its develop debuging line.
removed. https://github.com/google/google-api-ruby-client/pull/417/commits/f1f01125122216bb6685364a6bccd56ae23a2a77

**Pull request merged**